contrib/debian/control

```diff
@@ -0,0 +1,23 @@
+Source: netdata
+Build-Depends: debhelper (>= 9),
+ dpkg-dev (>= 1.13.19),
+ dh-systemd (>= 1.5),
+ zlib1g-dev,
+ autoconf,
+ automake,
+ pkg-config
+Section: net
+Priority: optional
+Maintainer: Costa Tsaousis <costa@tsaousis.gr>
```

Comment:
@lhw Agreed. Perhaps an obvious example name will do.

RE:
> I also think it would be easier to maintain the debian directory separately from the upstream repository and base it on the tags/releases instead of HEAD.

I've found it useful to use the main repository as a collaboration point to promote consistent packaging across distributions and to allow the user to easily generate their own personal packages, as seems to be the case here. Distribution packagers then have the option of using the example packaging as a basis for their own work, or not.

I've done this with [BWIPP](https://github.com/bwipp/postscriptbarcode) making it clear that the packaging is provided as a distribution-agnostic example:

https://github.com/bwipp/postscriptbarcode/tree/master/packaging-examples

I use OBS to check out of the main repository and build the example packaging into binaries for most major distributions on each commit (or tag) using a trigger as part of my Continuous Integration so that I know when I've broken something or made a packager's life difficult - in which case I fix things and provide heads up.

https://build.opensuse.org/package/show/home:terryburton:postscriptbarcode/libpostscriptbarcode

(A nit is that OBS makes it difficult to not have the debian packaging in the repository root as debian/ otherwise it would go in example-packaging/debian/ or at lease debian.upstream/. I'm working on a fix for this.)

I absolutely agree that the actual distribution-specific packaging should be hosted wherever is least-surprising to the community, e.g. Alioth, etc.